source/modulo_core/src/translators/message_readers.cpp

```diff
@@ -82,6 +84,41 @@ void read_message(state_representation::JointState& state, const sensor_msgs::ms
   }
 }
 
+void read_message(state_representation::JointTrajectory& state, const trajectory_msgs::msg::JointTrajectory& message) {
+  try {
+    state.set_name(message.header.frame_id);
+    if (!message.joint_names.empty()) {
+      state.set_joint_names(message.joint_names);
+    } else {
+      throw exceptions::MessageTranslationException("JointTrajectory message has no joint names");
```

Comment:
This might probably result in a exception message like `"MessageTranslationException: MessageTranslationException: JointTrajectory message has no joint names"` (not sure, could be tested). If yes maybe it would be neater to just use a std::exception

Reply:
yes, it would result to what you are saying, I was indeed wondering if exception is better here. Let me make the change, thanks